jkube-kit/build/api/src/main/java/org/eclipse/jkube/kit/build/api/auth/handler/SystemPropertyRegistryAuthHandler.java

```diff
@@ -22,44 +22,43 @@
 import org.eclipse.jkube.kit.build.api.auth.RegistryAuthHandler;
 import org.eclipse.jkube.kit.common.KitLogger;
 
-
 /**
  * @author roland
  */
 public class SystemPropertyRegistryAuthHandler implements RegistryAuthHandler {
 
-    private final RegistryAuthConfig registryAuthConfig;
```

Comment:
This class was completely reformatted and is really hard to track the changes (at least in the GH interface).
Could you please revert the reformatting so that it's clearer for us to see what was changed here?